apps/studio/src/modules/song/types.ts

```diff
@@ -156,69 +152,6 @@ export interface CloudinarySignatureResponse {
   readonly timestamp: number;
 }
 
-export enum MintingStatus {
```

Comment:
Did minting status come over due to the `Song` interface dependence, it won't be something we try and show on Marketplace, correct?